app/controllers/data_vis_controller.rb

```diff
@@ -1,5 +1,37 @@
 class DataVisController < ApplicationController
+    def data_view
+        #overall tour information
+        @tour_name = Tour.find_by(id: 7)&.name
+        @expenses_tour = Expense.where(tour_id: 7)
 
-    def data_view; end
-    
+        #budget calculations
+        tour_budget = Tour.find_by(id: 1)&.budget 
+        total_expenses_cost = Expense.where(tour_id: 1).sum(:cost)
```

Comment:
```suggestion
        total_expenses_cost = @expenses.where(tour_id: 1).sum(:cost)
```
You have already retrieved this data above